src/Raven.Embedded/EmbeddedServer.cs

```diff
@@ -211,8 +224,10 @@ private void ShutdownServerProcess(Process process)
                     {
                         inputStream.WriteLine("shutdown no-confirmation");
```

Comment:
@ayende This line on my Fedora workstation with .net 9.0 throws pretty much instantly but I think this might be expected. About to try this on a Windows Server box.

```txt
2025-02-19T13:04:10.9528086, 17, Information, Embedded, Raven.Embedded.EmbeddedServer, Failed to shutdown server PID 131422 gracefully in 00:00:30, EXCEPTION: System.IO.IOException: Pipe is broken.
   at System.IO.Pipes.PipeStream.CheckWriteOperations()
   at System.IO.StreamWriter.Flush(Boolean flushStream, Boolean flushEncoder)
   at System.IO.StreamWriter.Dispose(Boolean disposing)
   at System.IO.TextWriter.Dispose()
   at Raven.Embedded.EmbeddedServer.ShutdownServerProcess(Process process) in /_/62017/src/Raven.Embedded/EmbeddedServer.cs:line 212
```

Reply:
It may be needed if you are waiting for things to properly flush, I think